setup.ts

```diff
@@ -46,6 +48,68 @@ async function accessAndRefreshTokens(
   }
 }
 
+function transactionLogPath(logPath: string | null): void {
+  const config = dotenv.parse(fs.readFileSync(".env"));
+  if (!logPath) {
+    // Check if the logs/transaction.log file exists, if not, create it
+    const defaultLogPath = path.resolve(__dirname, "logs");
+    const defaultLogFile = path.join(defaultLogPath, "transaction.log");
```

Comment:
Notify the developer here that such a directory has been created here for logging.